products/workers/src/content/learning/using-durable-objects.md

```diff
@@ -457,14 +447,11 @@ export class Counter {
 }
 ```
 
-## Configuration Script
-
-While using Wrangler is strongly recommended, if you would really rather not use it for some reason we've included a [shell script](/publish-durable-object.sh) to automate the curl commands involved in uploading a Worker that implements and uses Durable Objects.
-
 ## Troubleshooting
 
 ### Debugging
-`wrangler dev` does not currently support Durable Objects.
+
+- [`wrangler dev`](/cli-wrangler/commands#dev) now supports Durable Objects.
```

Comment:
This looks weird -- a single bullet point mentioning `wrangler dev` without explaining it right next to a big paragraph explaining `wrangler tail`.